local-setup/jupyter/dataloader/kubectl_api.py

```
@@ -0,0 +1,245 @@
+    tenant_id = request.args.get('tenant_id')
+    env = request.args.get('env', 'chakshu')
+
+    if not tenant_id:
+        return jsonify({'error': 'tenant_id required'}), 400
+
+    try:
+        sql = f"""
+            SELECT
+                (SELECT COUNT(*) FROM boundary WHERE tenantid='{tenant_id}') as boundaries,
+                (SELECT COUNT(*) FROM boundary_relationship WHERE tenantid='{tenant_id}') as relationships;
+        """
```

Ruff (0.15.0)

[error] Possible SQL injection vector through string-based query construction

(S608)